README.md

```diff
@@ -1,3 +1,6 @@
+# DEPRECATION NOTICE
+Upcoming in Vault 1.14, we will stop publishing official Dockerhub images and publish only our Verified Publisher images. Users of Docker images should pull from hashicorp/vault instead of vault. Verified Publisher images can be found at https://hub.docker.com/r/hashicorp/vault.
```

Comment:
```suggestion
Upcoming in Vault 1.14, we will stop publishing official Dockerhub images and publish only our Verified Publisher images. Users of Docker images should pull from `hashicorp/vault` instead of `vault`. Verified Publisher images can be found at https://hub.docker.com/r/hashicorp/vault.
```

Reply:
⛏️ Perhaps make the refs code since it's the actual bits users need to update?